### ISSUE TITLE
Accessibility issue with table header: Required ARIA children role not present: cell, columnheader, rowheader, gridcell

### ISSUE THREAD

**Issue by markhowardt4** [jbetancur/react-data-table-component#610]
## Issue Check list
- [x] Agree to the [Code of Conduct](https://github.com/jbetancur/react-data-table-component/blob/master/CODE-OF-CONDUCT.md)
- [x] Read the README
- [x] You are using React 16.8.0+
- [x] You installed `styled-components`
- [x] Include relevant code or preferably a [code sandbox](https://codesandbox.io/embed/react-data-table-sandbox-ccyuu
)

## Describe the bug
The generated HTML has a role of  table, however the table header does not have a role of rowheader which is required to pass accessibility tests.

## To Reproduce
1. Using the example provided https://jbetancur.github.io/react-data-table-component/?path=/story/general--kitchen-sink
2. Analyze the page using the axe-core plugin for firefox
3. Notice the error Ensure elements with an ARIA role that require child roles contain them

## Expected behavior
The header column should have a role of rowheader

## Code Sandbox, Screenshots, or Relevant Code
https://github.com/jbetancur/react-data-table-component/blob/b9e6b888beb5c49b7655f7c6e637fb3ea858a327/src/DataTable/TableHeader.js#L38 could have the role rowheader instead of row

## Versions (please complete the following information)
 - React (RDT requires 16.8.0+) 16.13.1
 - Styled Components 5.1.0
 - OS: Linux
 - Browser Firefox
